wiki/Gameplay/Game_modifier/Perfect/zh.md

```diff
@@ -28,21 +28,21 @@ tags:
 
 ## 说明
 
-**Perfect** [游戏模组](/wiki/Gameplay/Game_modifier)通过在漏掉音符、[打击物件](/wiki/Gameplay/Hit_object)获得 300 以外的得分等等情况下，自动认定失败、重新开始[谱面](/wiki/Beatmap)，旨在帮助想要获得 SS 等级的玩家。
+**Perfect** [游戏模组](/wiki/Gameplay/Game_modifier)通过在漏掉音符、[打击物件](/wiki/Gameplay/Hit_object)获得 PERFECT 以外的得分等等情况下，自动认定失败、重新开始[谱面](/wiki/Beatmap)，旨在帮助想要获得 SS 等级的玩家。
 
 下列任一情况**会导致**此模组重新开始谱面：
 
 - 漏音符
-- 获得 `100` 或 `50`
+- 获得 `GOOD` 或 `MEH`
 - 未能完成转盘
 - 断滑条
 
 下列情况**不会导致**此模组重新开始谱面：
 
 - 获得 `300`
```

Comment:
shouldn't this be `GREAT` instead, for consistency?

Reply:
Nope, when you get a GREAT in lazer, you got a 300.

Reply:
![image](https://github.com/ppy/osu-wiki/assets/58957468/37539ae6-bee9-47df-be0a-c8bdfc7a7142)